fleet/mocks/agent.go

```diff
@@ -15,20 +15,20 @@ type agentRepositoryMock struct {
 	agentsMock map[string]fleet.Agent
 }
 
-func (a agentRepositoryMock) SetStaleStatus(ctx context.Context, minutes time.Duration) (int64, error) {
+func (a agentRepositoryMock) SetStaleStatus(_ context.Context, _ time.Duration) (int64, error) {
```

Comment:
was this stuff just to fix warnings?

Reply:
yes and also flag the filter of changes on fleet